samples/react/README.md

```diff
@@ -0,0 +1,85 @@
+This project integrates monaco-kusto in a create-react-app application.
+
+The following changes were made to a default create-react-app application:
+
+1. Used script tags to import Kusto language service (this piece of code is C# code transpiled to javascript. it has several issues that prevent it from being imported as a regular import statement).
```

Comment:
It is hard to understand in which file the change has been made. It would be a good idea to mention index.html. 
I would change "import Kusto language service" to "import Kusto language service and its dependencies (from `%PUBLIC_URL%/monaco-editor`)"